pkg/config/configobserver/configobserver_test.go

```diff
@@ -28,10 +28,10 @@ func Test_ConfigObserver_ChangeSupportConfig(t *testing.T) {
 	}{
 		{name: "interval too short",
 			config: map[string]*corev1.Secret{
-				pullSecretKey: &corev1.Secret{Data: map[string][]byte{
+				pullSecretKey: {Data: map[string][]byte{
 					".dockerconfigjson": nil,
 				}},
-				supportKey: &corev1.Secret{Data: map[string][]byte{
+				supportKey: {Data: map[string][]byte{
```

Comment:
Hmm I think I like the explicit definition more.

Reply:
I don't know. I think that, once the parent object is already a `corev1.Secret` by doing it two times, it is just overhead. So, IMHO, the change looks good.